azure-iot-device/tests/provisioning/shared_client_tests.py

```diff
@@ -18,9 +18,9 @@
 logging.basicConfig(level=logging.DEBUG)
 
 
-fake_provisioning_host = "hogwarts.com"
-fake_registration_id = "MyPensieve"
-fake_id_scope = "Enchanted0000Ceiling7898"
+fake_provisioning_host = "hostname.com"
+fake_registration_id = "MyRegId"
+fake_id_scope = "Some0000Scope7898"
 fake_symmetric_key = "Zm9vYmFy"
```

Comment:
In variables that have key in there name I would generally set them with a uuid (by calling a function).
MUCH less likely to get hit by password checked in tools.

Reply:
We have already set up a custom rule to ignore this specific key variable in our static analysis